build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '1.0.7'
+version '1.0.8'
```

Comment:
This will need to be 1.1 or 1.0.9 as 1.0.8 is taken now

Reply:
@JackReevies  - bumped to 1.0.9